buildSrc/src/main/java/Dependencies.kt

```diff
@@ -39,7 +39,7 @@ object Versions {
     // The version number below tracks the application-services version
     // that we depend on directly for tests, and it's important that it
     // be kept in sync with the version used by android-components above.
-    const val mozilla_appservices = "0.34.0"
+    const val mozilla_appservices = "0.35.0"
```

Comment:
bump to 0.35.4 (same for a-c pr)

Reply:
I'll wait for 0.35.5 which we're going to cut in a few minutes.

Reply:
And now it's 0.36!